packages/wallets/src/features/cashier/modules/TransactionStatus/utils/transactionDisplayMapper.ts

```diff
@@ -0,0 +1,130 @@
+import moment from 'moment';
+import { getTruncatedString } from '@deriv/utils';
+import { THooks } from '../../../../../types';
+
+const transactionDetailsMapper = (transaction: THooks.CryptoTransactions) => {
+    const transactionHashObscure = transaction.transaction_hash
+        ? `${getTruncatedString(transaction.transaction_hash, { length: 8, type: 'middle' })}`
+        : 'Pending';
+
+    if (transaction.transaction_type === 'deposit') {
+        return {
+            CONFIRMED: {
+                confirmationDisplay: 'Confirmed',
+                description: 'Your deposit is successful.',
+                statusColor: 'successful',
```

Comment:
that sounds like a styling concern,
wouldn't be better to add modifiers like:

.transaction-status--confirmed, 
.transaction-status--error,
.transaction-status--pending,
....

Reply:
Why not use CONFIRMED, PENDING, ERROR, etc directly as modifier and skip the "mapper" completely? :)
one less thing to learn&memorise, one mental "jump" less for any future code readers :)

And the naming actually gonna be less confusing - as it means, that across codebase, we consistently use domain-based naming. Without any extra mappings "on the way". Meaning that we use the same enums on the backend, api, javascript and css :) And then all the extra mappings can be removed entirely.

Reply:
Ah, got it, this is a good idea, I've made the changes 👍 
The only downside is when the API changes, and so do `api-types`, TypeScript raises errors in the mapper, but how would a dev know that CSS also needs to be updated with the latest API types? 🤔

Reply:
I think it should work the same way as with mapping - if backend returns something unexpected/not supported by frontend, use default style/color,